_maps/cyberiad.dm

```diff
@@ -13,14 +13,16 @@ z7 = empty
 */
 
 #if !defined(USING_MAP_DATUM)
-	#include "map_files\cyberiad\cyberiad.dmm"
-	#include "map_files\cyberiad\z2.dmm"
-	#include "map_files\cyberiad\z3.dmm"
-	#include "map_files\cyberiad\z4.dmm"
-	#include "map_files\generic\z5.dmm"
-	#include "map_files\cyberiad\z6.dmm"
-	#include "map_files\generic\z7.dmm"
```

Comment:
Why did you uppercase all of these again?

Reply:
Dont fully remember a reason. If there is one then its probably for consistency or something

Reply:
Fixed